source/utils/generate-stack.ts

```diff
@@ -0,0 +1,19 @@
+import isNode from './node/is-node';
+
+/**
+Generates a useful stacktrace that points to the user's code where the error happened,
+on platforms without the `Error.captureStackTrace` function present
+
+@hidden
+*/
+export const generateStackTrace = () => {
+	let stack = new RangeError('INTERNAL_OW_ERROR').stack!;
+
+	// If we're running in a node environment without the `Error.captureStackTrace` function present
```

Comment:
Is this even possible? AFAIK the Node.js versions we target support this method.

Reply:
Sanity checks if someone or something overrides or removes it. Doesn't hurt to have truthfully

Reply:
Then I would rather just crash hard. It is a Node.js API and expected to be there when in Node.js. We don't gracefully handle `fs.readFileSync`, etc., being overridden either.

Reply:
While I agree, and I could remove this if check, I'd suggest we keep it. `generateStackTrace` is a fallback for any env that doesn't use v8 (Firefox or Safari for example, since you provide browser builds) or that doesn't have `captureStackTrace` present (aforementioned browsers). It..doesn't hurt to keep it, but it is up to you 👍

Reply:
Eh, guess you're right, the chances of the API being missing in node is small. Will remove